salt/auth/django.py

```diff
@@ -95,21 +111,24 @@ def django_auth_setup():
         django_model_name = django_model_fullname.split('.')[-1]
         django_module_name = '.'.join(django_model_fullname.split('.')[0:-1])
 
-        __import__(django_module_name, globals(), locals(), 'SaltExternalAuthModel')
+        django_auth_module = __import__(django_module_name, globals(), locals(), 'SaltExternalAuthModel')
         DJANGO_AUTH_CLASS_str = 'django_auth_module.{0}'.format(django_model_name)
         DJANGO_AUTH_CLASS = eval(DJANGO_AUTH_CLASS_str)  # pylint: disable=W0123
 
-    if django.VERSION >= (1, 7):
-        django.setup()
-
 
 def auth(username, password):
     '''
     Simple Django auth
     '''
-    import django.contrib.auth  # pylint: disable=import-error
+    sys.path.append(__opts__['django_auth_path'])
```

Comment:
@hujunya On closer review I think this will be OK, but could you check `sys.path` before appending to it just to make sure we don't add `django_auth_path` multiple times? I don't think there is any check in the Python runtime that would keep us from doing that.